packages/core-components/src/components/SupportButton/SupportButton.tsx

```diff
@@ -53,10 +53,16 @@ const useStyles = makeStyles(
   { name: 'BackstageSupportButton' },
 );
 
-const SupportIcon = ({ icon }: { icon: string | undefined }) => {
+const SupportIcon = ({ icon }: { icon: string | JSX.Element | undefined }) => {
```

Comment:
There is no strict decision on it, but it would be nice to make this an `IconComponent` to align with the Icon types for Scaffolder & Sidebar.

Reply:
I tried that first, but I couldn't figure out how to make the code 

```
const Icon = (props: SvgIconProps) =>
      React.createElement(
        SvgIcon,
        props,
        <path d="M9 7H4a2 2 0 1 0 0 4h5a2 2 0 1 0 0-4zm2-3v2H9a2 2 0 1 1 2-2zM7 14v5a2 2 0 1 0 4 0v-5a2 2 0 1 0-4 0zm-3-2h2v2a2 2 0 1 1-2-2zm10 4h5a2 2 0 1 0 0-4h-5a2 2 0 1 0 0 4zm-2 3v-2h2a2 2 0 1 1-2 2zm4-10V4a2 2 0 1 0-4 0v5a2 2 0 1 0 4 0zm3 2h-2V9a2 2 0 1 1 2 2z" />,
      );
```

be accepted as an IconComponent

Reply:
Noting that as discussed we prefer `IconComponent` here for consistency